packages/extension-sample/src/types/config.ts

```diff
@@ -4,9 +4,14 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import {
+    ApplicationExtensionConfig
+} from '@salesforce/pwa-kit-runtime/ssr/server/extensibility'
```

Comment:
The the problem I eluded to earlier is happening right here. We have a singular configuration type for the extensions exported, but it's extending from the `server/extensibility` location from the runtime. 

You can see the problem is that we are using that in `setup-app` which would be getting it's configuration from the `react-sdk`.